src/ucp/wireup/select.c

```diff
@@ -828,7 +828,7 @@ static int ucp_wireup_compare_score(const void *elem1, const void *elem2,
     score2 = (*lane2 == UCP_NULL_LANE) ? 0.0 : lanes[*lane2].score[lane_type];
 
     /* sort from highest score to lowest */
-    return (score1 < score2) ? 1 : ((score1 > score2) ? -1 : 0);
+    return -ucp_score_cmp(score1, score2);
```

Comment:
```suggestion
    return ucp_score_cmp(score2, score1);
```

Reply:
IMO "-" is looks clearer vs swapping args ordering

Reply:
IMO the `-` is indeed a bit clearer, but it doesn't really matter, as the reader may overlook it in both cases.
Perhaps more important is to clarify the comment above so that it explains it explicitly.

Reply:
there is a comment just line above about sorting ordering

Reply:
I meant making that comment more explicit.